source/directives/uiTreeNode.js

```diff
@@ -116,10 +117,12 @@
               tagName = scope.$element.prop('tagName');
 
               if (tagName.toLowerCase() === 'tr') {
-                placeElm = angular.element($window.document.createElement(tagName));
-                tdElm = angular.element($window.document.createElement('td'))
+                placeElm = angular.element($window.document.createElement(tagName))
                   .addClass(config.placeholderClass);
-                placeElm.append(tdElm);
+                for (i = 0; i < scope.$element[0].children.length; i++) {
```

Comment:
Hey @Voles, @mariocasciaro , you actually need to specify the colspan attribute on the td element. That way, the placeholder will always take full width of the tbody with no need of for loop. 
We also need a `display: table-row;` on the `.angular-ui-placeholder` class.
```javascript
placeElm = angular.element($window.document.createElement(tagName))
   .addClass(config.placeholderClass)
tdElm = angular.element($window.document.createElement('td'))
   .attr('colspan', scope.$element[0].children.length);
placeElm.append(tdElm);
```
Cheers,
Leo.